package.json

```diff
@@ -7,6 +7,7 @@
   "type": "module",
   "exports": {
     ".": {
+      "types": "./dist/index.d.ts",
```

Comment:
If you have more context of why this is required and how its different from https://github.com/kevinmarrec/h3-typebox/blob/a6f88f08961dff74bb87ff7a1d3816b219c58cf3/package.json#L16 ? Thanks !

Reply:
Nodenext/node16 model resolution switched to this property instead of the one in the root at some Typescript version update - having both allows for newer and older consumers to have proper type resolution. See: https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing

Reply:
Makes sense, thank you !